templates/download/iot/installation-medias.html

```diff
@@ -0,0 +1,239 @@
+{% extends "download/_base_download.html" %}
```

Comment:
This page, "installation-medias.html" should presumably be named "installation-media.html". The word "media" is already a plural. ["Medium"](https://en.oxforddictionaries.com/definition/medium) is the singular.